tests/components/BurgerMenuNavCategories.test.js

```diff
@@ -0,0 +1,305 @@
+import { shallowMount } from '@vue/test-utils'
+import BurgerMenuCategories from '~/components/BurgerMenuNavCategories.vue'
+import PixPrismicLink from '~/components/PixPrismicLink.vue'
+
+describe('Component: Burger Menu Nav Categories', () => {
+  let component
+  const $t = () => {}
+
+  beforeEach(() => {
+    component = shallowMount(BurgerMenuCategories, {
+      components: { PixPrismicLink },
+      propsData: {
+        items: [
+          {
+            categoryName: [],
+            descriptionCategoryTitle: [],
+            descriptionCategoryText: [],
+            menuSectionTitle: [],
+            menuName: [
+              {
+                text: 'Accueil',
+              },
+            ],
+            beforeIcon: 'icn-home.svg',
+            alternativeTextForBeforeIcon: [{ text: 'Accueil' }],
+            afterIcon: null,
+            alternativeTextForAfterIcon: [],
+            menuLink: {
+              url: 'https://pro.pix.fr',
+            },
+            separator: false,
+          },
+          {
+            categoryName: [
+              {
+                text: 'Découvrir Pix',
+              },
+            ],
+            descriptionCategoryTitle: [
+              {
+                text: "Pix, c'est quoi ?",
+              },
+            ],
+            descriptionCategoryText: [
+              {
+                text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+              },
+            ],
+            menuSectionTitle: [
+              {
+                text: 'La plateforme Pix',
+              },
+            ],
+            menuName: [
+              {
+                text: 'Défis et compétences',
+              },
+            ],
+            beforeIcon: null,
+            alternativeTextForBeforeIcon: [],
+            afterIcon: null,
+            alternativeTextForAfterIcon: [],
+            menuLink: {
+              url: 'https://pix.fr/les-tests',
+            },
+            separator: false,
+          },
+        ],
+      },
+      stubs: { fa: true, 'burger-menu-nav-sections': true },
+      mocks: { $t },
+    })
+  })
+
+  afterEach(() => {
+    component.destroy()
+  })
+
+  it('should initially renders with no open category', () => {
```

Comment:
nitpick de conjugaison : should est le verbe conjugué de la phrase donc render ne prend pas de S, ça s'applique aussi aux autres titres de test de ce commit